src/main/java/com/j256/ormlite/dao/CloseableSpliterator.java

```diff
@@ -0,0 +1,20 @@
+package com.j256.ormlite.dao;
+
+import java.util.Spliterator;
+
+/**
+ * Extension to Spliterator to provide a close() method.
+ *
+ * <p>
+ * <b>NOTE:</b> You must call {@link CloseableSpliterator#close()} method when you are done otherwise the underlying SQL
+ * statement and connection may be kept open.
+ * </p>
+ */
+public interface CloseableSpliterator<T> extends Spliterator<T>, AutoCloseable {
+
+    /**
+     * Close any underlying SQL statements but swallow any SQLExceptions.
+     */
+    void closeQuietly();
```

Comment:
Tabs here please to match the other code and (to be pedantic) public.  Yes I know it isn't required but that also matches the other code.